R/ResampleDesc.R

```diff
@@ -141,6 +149,53 @@ makeResampleDescRepCV = function(reps = 10L, folds = 10L) {
   makeResampleDescInternal("repeated cross-validation", iters = folds * reps, folds = folds, reps = reps)
 }
 
+makeResampleDescFixedCV = function(horizon = 1L, initial.window = .5, size = 100L, skip = .01  ) {
+  horizon = asInteger(horizon, lower = 1L, upper = Inf)
+  assertNumeric(initial.window, lower = 1E-15, upper = 1)
+  size = asInteger(size, lower = 1L, upper = Inf)
+  assertNumeric(skip, lower = 0L, upper = Inf)
+  initial.window.desc = floor(initial.window * size)
```

Comment:
(The following is moot since you'll have to remove `size` anyways, but in case you are going to cut/paste this...)

If `initial.window` is small enough, this will be 0 and line 159 will give a hard to understand error. You should probably do either `ceiling` here, or check for `initial.window.desc > 0` and throw a meaningful error otherwise.

Also: What happens when `size - horizon` is smaller than `initial.window.desc`? Is it an error? Should the user be warned about that?